webapi/src/main/scala/org/knora/webapi/messages/admin/responder/projectsmessages/ProjectsMessagesADM.scala

```diff
@@ -427,7 +428,7 @@ object ProjectIdentifierADM {
  * @param size      the restricted view size.
  * @param watermark the watermark file.
  */
-case class ProjectRestrictedViewSettingsADM(size: Option[String] = None, watermark: Option[String] = None)
+case class ProjectRestrictedViewSettingsADM(size: Option[String] = None, watermark: Boolean)
```

Comment:
How about to apply `watermark` default value here instead of everywhere else?

Reply:
IMHO the default should only be decided in the cases where it makes sense, i.e. on the api level should the user not provide a value. 

I have even removed the size default here as it was unused. => cd952cd40f5b004ef02ef7f164e76e5c8c7ae8bd